scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -42,6 +42,9 @@
 - name: ADC_IN / ADC_INN / ADC_INP
   match: "^ADC(?:\\d+)?_IN[NP]?\\d+$"
 
+- name: ADC_VINM / ADC_VINP
+  match: "^ADC(?:\\d+)?_VIN[PM]\\d+$"
```

Comment:
Are we sure this won't have impact on other series ?
If this is the case, please split in another PR

Reply:
I have done a blank `genpinctrl` run with latest STM32_open_pin_data and did not see `ADC_VINM / ADC_VINP` in any pin control file other than WB0, as expected, so I don't think it has any impact.